tests/test_rigid_physics.py

```diff
@@ -793,7 +1012,7 @@ def test_pd_control(show_viewer):
     robot.set_dofs_armature(robot.get_dofs_armature(envs_idx=1) + MOTORS_KD * scene.sim._substep_dt, envs_idx=1)
 
     for i in range(1000):
-        dofs_pos = robot.get_dofs_position(envs_idx=1)
+        dofs_pos = robot.get_qpos(envs_idx=1)
```

Comment:
Note that get_dofs_position returns the Euler angles for rotational DOFs, while qpos is a quaternion. Want to make sure this change is intentional?

Reply:
Yes it is ! This change is necessary because qpos gets updated at after integration but dof vel no longer does :)

Reply:
(Not that it is better this way, the less is updated post-integration, the better. For consistency of the data.